examples/dreambooth/train_dreambooth_lora_sd3.py

```diff
@@ -1687,7 +1687,7 @@ def get_sigmas(timesteps, n_dim=4, dtype=torch.float32):
                 if accelerator.sync_gradients:
                     params_to_clip = itertools.chain(
                         transformer_lora_parameters,
-                        text_lora_parameters_one,
+                        text_lora_parameters_one if args.train_text_encoder else transformer_lora_parameters,
                         text_lora_parameters_two if args.train_text_encoder else transformer_lora_parameters,
                     )
```

Comment:
We don't need to chain anything if `args.train_text_encoder` is False. 

Could you edit this such that it becomes this?

https://github.com/huggingface/diffusers/blob/8690e8b9d6e68b3aec28f7f84374dd8256e79f18/examples/dreambooth/train_dreambooth_lora_sdxl.py#L1810